Formula/spek.rb

```diff
@@ -0,0 +1,31 @@
+class Spek < Formula
+  desc "Acoustic spectrum analyser"
+  homepage "https://www.spek.cc"
+  url "https://github.com/alexkay/spek/releases/download/v0.8.5/spek-0.8.5.tar.xz"
+  sha256 "1bccf85a14a01af8f2f30476cbad004e8bf6031f500e562bbe5bbd1e5eb16c59"
+  license "GPL-3.0-only"
+
+  depends_on "pkg-config" => :build
+
+  depends_on "ffmpeg"
+  depends_on "wxwidgets"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}",
+                          "--libdir=#{lib}",
+                          "--disable-silent-rules"
+    system "make"
+
+    if OS.linux?
+      # https://github.com/alexkay/spek/issues/235
+      cp "data/spek.desktop.in", "data/spek.desktop"
+    end
+
+    system "make", "install"
+  end
+
+  test do
+    assert_match "Spek version #{version}", shell_output("#{bin}/spek --version") if OS.mac?
```

Comment:
Maybe this is better?
```suggestion
    return if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"].present?

    assert_match "Spek version #{version}", shell_output("#{bin}/spek --version")
```
That way, the tests can be run locally on Linux.